src/Authentication/Authentication/custom/Find-MgGraphCommand.ps1

```diff
@@ -187,7 +187,10 @@ Function Find-MgGraphCommand {
                 [Parameter(Mandatory = $true, Position = 0)]
                 [System.Uri]$Uri
             )
-
+            #Check if Uri contains /me and replace it with /users/{user-id}
+            if ($Uri.AbsoluteUri.Contains("/me/")) {
```

Comment:
I think it's better to use a relative to make the Uri relative to the graph api root and check `$Uri.StartsWith("/me/)`. That way, you'll not have a problem with urls like `https://graph.microsoft.com/v1.0/profile/me/` if it ever existed in future.